packages/stockflux-launcher/src/titlebar/Titlebar.js

```diff
@@ -0,0 +1,68 @@
+import cx from 'classnames';
+import React, { useCallback } from 'react';
+import { OpenfinApiHelpers } from 'stockflux-core';
+import Components from 'stockflux-components';
+import './Titlebar.css';
+
+export default ({ dockedTo }) => {
+  const onMinimizeClick = useCallback(async () => {
+    const currentWindow = await OpenfinApiHelpers.getCurrentWindow();
+    currentWindow.minimize();
+  }, []);
+
+  const onCloseClick = useCallback(async () => {
+    const currentWindow = await OpenfinApiHelpers.getCurrentWindow();
+    currentWindow.close();
+  }, []);
+
+  const dockedHorizontal = dockedTo === 'top' || dockedTo === 'none';
+
+  return (
+    <div className="titlebar">
+      {dockedHorizontal && (
+        <div
+          className={cx(
+            'column column-left',
+            dockedTo === 'top' || dockedTo === 'none' ? 'add-padding-left' : ''
+          )}
+        >
+          <p className="title">
+            <span className="title-strong">Stock</span>
+            <span className="title-light">Flux</span>
+          </p>
+        </div>
+      )}
+
+      <div
+        className={cx(
+          'column column-right header-icon',
+          dockedTo === 'top' || dockedTo === 'none' ? 'add-padding-right' : ''
+        )}
+        onClick={onCloseClick}
+        title="Close"
+      >
+        <Components.Icons.Toolbar.CloseIcon />
+      </div>
+
+      <div
```

Comment:
These divs seem similar as well, could be combined into it's own component or a function that returns a JSX snippet.

Reply:
Also, I think this should use titlebar from `stockflux-components` rather than define it's own. The one in `stockflux-components` can be amended to accept JSX `<p><strong>Stock</strong>Flux</p>` as `children`.